hermes/probe/probe.go

```diff
@@ -107,17 +100,18 @@ func (mp *Probe) Init(name string, opts *options.Options) error {
 // This is a required method to implement the cloudprober.Probes.Probe interface.
 // Arguments:
 //	- ctx: context provided for cancelling probe.
-//	- metricChan: unidirectional channel used for sending metrics to be surfaced.
-func (mp *Probe) Start(ctx context.Context, metricChan chan<- *cpmetrics.EventMetrics) {
-	probeTicker := time.NewTicker(mp.interval())
+//	- metricChan: bidirectional channel used for sending metrics to be surfaced.
+//		- Must be bidirectional to satisfy cloudprober.Probes.Probe interface.
+func (p *Probe) Start(ctx context.Context, metricChan chan *cpmetrics.EventMetrics) {
+	probeTicker := time.NewTicker(p.interval())
```

Comment:
Nit: No need to include "probe" in the variable name, as there's only one ticker. Just `ticker` would do.

Reply:
Okay, I've changed it to `ticker`.

Reply:
Hmm it still appears as `probeTicker` for me. Can you double-check you committed/amended the right version?

Reply:
I had made the change locally but not committed yet. I've pushed the commit now and it's displaying correctly for me on GitHub.